book/security.rst

```diff
@@ -1352,6 +1352,15 @@ FriendsOfPHP organization.
     any of your dependencies is affected by a known security vulnerability.
     Therefore, you can easily integrate it in your build process.
 
+.. note::
+
+    To enable the ``security:check`` command, you need to add the
+    `sensio distribution bundle`_ in your composer dependencies.
```

Comment:
If you see other links with references to bundles the formatting is not CamelCase. See line 1391 for the frameworkextrabundle